GOVERNANCE.md

```diff
@@ -108,10 +198,11 @@ In some cases, a supermajority vote is required for decision making - at least t
 Some examples include:
 
 * Carrying out [Code of Conduct](https://oscal-compass.github.io/compliance-trestle/mkdocs_code_of_conduct/) decisions requiring severe censure (supermajority of the Oversight Committee)
```

Comment:
Supermajority is not defined. I believe you are saying 2/3rds but it should be explicit